Core/src/org/sleuthkit/autopsy/datamodel/ContentTagNode.java

```diff
@@ -123,13 +128,14 @@ protected Sheet createSheet() {
         List<Action> actions = new ArrayList<>();
         actions.addAll(Arrays.asList(super.getActions(context)));
 
-        AbstractFile file = getLookup().lookup(AbstractFile.class);
+        AbstractFile file = getLookup().lookup(AbstractFile.class
+        );
```

Comment:
One line? Though I'm being really nitpicky.

Reply:
Oops thank you for catching this I will get it fixed. I must of auto formatted with a misplaced bracket or paren at some point, it tends to screw up stuff like this and the auto format doesn't correct it when done again after the bracket/paren is restored.